websites/A/Astr.io/dist/metadata.json

```diff
@@ -16,8 +16,8 @@
     "ga_IE": "Cluiche brabhsálaí gréasáin atá saor in aisce a sheinntear mar a imríonn úsáideoirí mar phláinéid agus a scriosann daoine eile!"
   },
   "url": "astr.io",
-  "version": "1.2.4",
-  "logo": "https://i.imgur.com/jgsd4vO.png",
+  "version": "1.2.5",
+  "logo": "https://i.imgur.com/wsU707Y.png",
```

Comment:
upload proof that it changed in the presence itself too

Reply:
what?? it didnt change in the presence, only reason it changes in metadata is because the image vanished from imgur...... it was just a reupload fixing a 404